### PR TITLE
MET-32 Update docs

### PR DESCRIPTION
**Pull request by dariacm**

### CODE REVIEW

File: packages/amqp/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# AMQP (Advanced Message Queuing Protocol)
+
```
Comment (kibertoad) [inline packages/amqp/README.md:2]: can you also add link to https://github.com/kibertoad/message-queue-toolkit/blob/main/README.md for transport-agnostic library documentation?

**Comment by kibertoad** [comment]
Can you also mention that both direct exchanges (via AbstractAmqpQueuePublisher/AbstractAmqpQueueConsumer) and topic exchanges (via AbstractAmqpTopicPublisher and AbstractAmqpTopicConsumer) are supported?

**Review by kibertoad — APPROVED** [review]

File: packages/amqp/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+The library provides support for both direct exchanges and topic exchanges.
+
+> **_NOTE:_**  Check [README.md](../../README.md) for transport-agnostic library documentation.
+>
+
+## Publishers
+
+Use `AbstractAmqpQueuePublisher` to implement direct exchange and `AbstractAmqpTopicPublisher` for topic exchange.
```
Comment (kibertoad) [inline packages/amqp/README.md:10]: we also need to document out PublisherManager functionality, but that's a separate big topic

Comment (dariacm) [reply packages/amqp/README.md:10]: I've created a separate ticket to not stall this one -> https://lokalise.atlassian.net/browse/MET-53

**Pull request merged**